terraform/network.tf

```diff
@@ -3,7 +3,7 @@ data "aws_availability_zones" "available" {
 
 resource "aws_vpc" "main" {
   cidr_block           = "10.0.0.0/16"
-  enable_dns_hostnames = false
+  enable_dns_hostnames = true
```

Comment:
This is required for the ECR Interface endpoint
![image](https://user-images.githubusercontent.com/32230328/110785045-5d615d80-8262-11eb-812d-947e86b951bb.png)

More info here: https://aws.amazon.com/blogs/compute/setting-up-aws-privatelink-for-amazon-ecs-and-amazon-ecr/